### ISSUE TITLE
Prompt overwritten after command not found

### ISSUE THREAD

**Issue by alcoheca** [sorin-ionescu/prezto#578]
Using terminator, and this happens after installing a fresh copy:

![zsh-probs](https://f.cloud.github.com/assets/546879/2446261/b7ae38dc-ae82-11e3-9c63-3af2149363ff.png)

Basically after running a command which is unknown, the position of the cursor jumps left 2 spaces so that it overwrites the coloured prompt.

**Comment by sirplatypos** [comment]
sorry to bump a dead thread, but I am now having the exact same problem, except I am using termite and when I tried xterm the same problem occurred.

**Comment by belak** [comment]
Both termite and terminator use libvte, so this doesn't surprise me too much. I'd imagine xterm is having issues because of unicode issues.

What prompt are you using?

**Comment by sirplatypos** [comment]
I modified the sorin prompt, but after this problem started I’ve done everything including reverting the file back to its original state, which hasn’t solved the problem. I have recently run into some field being corrupted after I was forced to manually run fsck, and at this point I am convinced that prezto is probably not at fault, especially since the problem does not occur on my laptop.
I now think that a corrupted config file is to blame (but sadly have absolutely no idea which one or where to look).  I modified the prompt so that it does not actually contain anything that is not ascii to my knowledge, so I would not think that was the problem.

Thank you for the reply, but at this point I am led to believe that a config file got corrupted. I will update and test my laptop for the issue, and let you know if I find anything significant.

Sent from my iPhone

On Jul 2, 2018, at 8:15 PM, Kaleb Elwert <notifications@github.com<mailto:notifications@github.com>> wrote:


Both termite and terminator use libvte, so this doesn't surprise me too much. I'd imagine xterm is having issues because of unicode issues.

What prompt are you using?
